src/FakeItEasy/SelfInitializedFakes/CallData.cs

```diff
@@ -23,15 +21,14 @@ public class CallData
         public CallData(MethodInfo method, IEnumerable<object> outputArguments, object returnValue)
         {
             this.Method = method;
-            this.OutputArguments = outputArguments == null ? null : outputArguments.ToArray();
+            this.OutputArguments = outputArguments.ToArray();

```

Comment:
Did you mean to write `outputArguments?.ToArray()`? I assume that if there was a null check, it's because `outputArguments` can be null...